win32/Lib/mmsystem.py

```diff
@@ -1,3 +1,5 @@
+import win32api
+
 # Generated by h2py from d:/msdev/include/mmsystem.h
```

Comment:
I'd be curious in how to regenerate this file. It could give more insights on why there's undefined names here.